docs/src/snap/howto/install/install-offline.md

```diff
@@ -0,0 +1,374 @@
+# Installing Canonical Kubernetes Offline or in an air gapped environment
+
+There are situations where it is necessary or desirable to run Canonical 
+Kubernetes on a machine that is not connected to the internet. 
+Based on different degrees of separation from the network,
+different solutions are offered to accomplish this goal.
+This guide explains the necessary preparation required for the
+offline installation and walks you through the different potential scenarios.
+
+## Prepare for Deployment
+
+In preparation for the offline deployment you will download the Canonical
+Kubernetes snap, fulfill the networking requirements based on your scenario and
+handle images for workloads and Canonical Kubernetes features.
+
+### Prep 1: Download the Canonical Kubernetes snap
+
+From a machine with access to the internet download the following:
+
+```bash
+sudo snap download k8s --channel 1.30-classic/beta
```

Comment:
link to the explanation page about which channel/revision to pick

add a `--basename k8s` flag and remove the `mv` commands below.